clang/test/CXX/over/over.load/p2-0x.cpp

```diff
@@ -24,11 +24,6 @@ class Y {
   void k() &&; // expected-error{{cannot overload a member function with ref-qualifier '&&' with a member function without a ref-qualifier}}
 };
 
-struct GH76358 {
```

Comment:
Why remove this test?

Reply:
Completely accidental, i did not notice...